### PR TITLE
add launch file demo_mode

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

**Comment by jlunenburg** [comment]
I added a concise README and comments referring to this in the launch file to clarify which of the three demo launch files to use in which situation.

@reinzor as the maintainer of this package: could you review this pull request?

File: launch/state_machines/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This folder contains three launch files that are useful when giving demos:
+* free_mode.launch: this launches ED with localization. Therefore, it entirely depends on the environment in which this is used.
```
Comment (LoyVanBeek) [inline launch/state_machines/README.md]: What does this mean? Can I use this in a random place or on in a known place?

**Review by LoyVanBeek — CHANGES_REQUESTED** [review]

File: launch/state_machines/demo_mode.launch [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```
Comment (jlunenburg) [inline launch/state_machines/demo_mode.launch]: Maybe demo_extern is more consistent

**Comment by MatthijsBurgh** [comment]
@LoyVanBeek please review again.

**Review by LoyVanBeek — APPROVED** [review]

**Comment by MatthijsBurgh** [comment]
@Rayman Do you agree with  b0c4004?

**Pull request merged**